### ISSUE TITLE
API Return Status should be json format

### ISSUE THREAD

**Issue by adarshtri** [Ayushi61/Classroom-Bot#67]
The current returns from API hits to the bot server return true or false. We understand this is something the team should have worked from start but nevertheless changes have to be made so the server returns JSON with proper status codes and all the services consuming those apis should be able to accommodate those changes.

**Comment by RSMuthu** [comment]
So basically, the need is to change all the api with return type as Boolean into a json return type,
and handle wherever needed on its internal calls.

Shall i take this up ?

**Comment by adarshtri** [comment]
Yes sure. Fork the project and go ahead with your changes. We would be happy to see your PR on the project.

**Comment by RSMuthu** [comment]
Thanks @adarshtri . As said,

`So basically, the need is to change all the api with return type as Boolean into a json return type,
and handle wherever needed on its internal calls.`

I shall try doing it ..
